### PR TITLE
Fourre-tout de release

### PR DESCRIPTION
**Pull request by FrancoisDevemy5**
Des petits sujets indépendants pour finaliser la release.
Chaque commit est isolé et la relecture se fera surement mieux commit par commit.

Certains important pour la release : 
- [Passage en version 6](https://github.com/ReachFive/identity-ios-sdk/pull/137/commits/c4e7864d52f109a41abd710ec13be9fbf8d8e67e)
- [Changelog](https://github.com/ReachFive/identity-ios-sdk/pull/137/commits/75c7543ca96fef3b3c195f9bb80f95407861f391)

Correction de bug : 
- [Meilleure gestion des erreurs de conf WeChat](https://github.com/ReachFive/identity-ios-sdk/pull/137/commits/32eb43d644d80c7083bf32aeb09ef0c575c9e524)

Breaking Change : 
- [Suppression d'une vieille méthode](https://github.com/ReachFive/identity-ios-sdk/pull/137/commits/920ea37d2c2fd16ad712055e4cdd8e5700e0eac0)
- [Passage en Future](https://github.com/ReachFive/identity-ios-sdk/pull/137/commits/9f14998aabcf669cb0fb9e69ba1e19570504b718)

Alignement du code avec la doc : 
- [Initialisation du SDK](https://github.com/ReachFive/identity-ios-sdk/pull/137/commits/e417a98adaf990cfadd01a695fe21d12c46b32bb)

### CODE REVIEW

**Comment by FrancoisDevemy5** [comment]
@talking-tortoise @jcbee could you please review the [Changelog](https://github.com/ReachFive/identity-ios-sdk/pull/137/commits/adff4d540ccc80c07a4663d1c8adeb0f0ccefdcf) to better align it with what will be in the release notes and the doc?

**Review by nbrr — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## v6.0.0
+
+Warning: There are Breaking Changes
+
+### New features
+- New module [IdentitySdkWeChat](IdentitySdkWeChat) for native WeChat Login Integration
```
Comment (jcbee) [inline CHANGELOG.md:8]:
```suggestion
- New module [IdentitySdkWeChat](IdentitySdkWeChat) for native WeChat Login integration 
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## v6.0.0
+
+Warning: There are Breaking Changes
+
+### New features
+- New module [IdentitySdkWeChat](IdentitySdkWeChat) for native WeChat Login Integration 
+- Add optional `origin` parameters to most authentification functions to help categorize your traffic
```
Comment (jcbee) [inline CHANGELOG.md:9]:
```suggestion
- Add optional `origin` parameters to most authentification functions to help categorize your inbound traffic
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,33 @@
...
+
+### Bug fix
+- Fix passwordless flow with email
+
+### Breaking changes
+ - The functionality of module `IdentitySdkWebView` has been integrated into the main `IdentitySdkCore` module. Therefore `IdentitySdkWebView` has been deprecated.
```
Comment (jcbee) [inline CHANGELOG.md:15]:
```suggestion
- The functionality of module `IdentitySdkWebView` has been integrated into the main `IdentitySdkCore` module. Therefore `IdentitySdkWebView` has been deprecated.
```

Comment (jcbee) [reply CHANGELOG.md:15]: > Therefore `IdentitySdkWebView` has been deprecated.

Just deprecated? Or removed?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,33 @@
...
+- The `origin` parameter in `NativeLoginRequest`, `NewPasskeyRequest`, `PasskeySignupRequest` has changed meaning.
+  - Up to now it meant the origin of the webauthn call. There is a new parameter `originWebAuthn` for this
+  - Now `origin` has the same meaning as in all other authentification functions: to help categorize your traffic
```
Comment (jcbee) [inline CHANGELOG.md:18]:
```suggestion
  - Now `origin` has the same meaning as in all other authentification functions: to help categorize your inbound traffic
```

File: CHANGELOG.md (hunk 3 above)
Comment (FrancoisDevemy5) [reply CHANGELOG.md:15]: It will still be available in versions up to 5.8.0

**Pull request merged**